packaging/install.cmake

```diff
@@ -110,6 +113,7 @@ if(UNIX)
   # Add header comment to the variable definition (if any variable is defined)
   if(ADDITIONAL_ENV_VAR)
     set(ADDITIONAL_ENV_VAR "\n# Additional variables for MPI\n${ADDITIONAL_ENV_VAR}")
+    set(ADDITIONAL_ENV_VAR_DISPLAY "\n\t# Additional variables for MPI\n${ADDITIONAL_ENV_VAR_DISPLAY}")
```

Comment:
Remove `\t` as `@ADDITIONAL_ENV_VAR_DISPLAY` is already indented in `khiops_env.in`.

Reply:
no there is a `\n` before the `\t`

Reply:
Yes, but I think the `\t` is added with respect to the whole `@ADDITIONA_ENV_VAR_DISPLAY@` block which is indented. Here:

- on Ubuntu 22.04 Docker: 
```bash 
$ cat $(which khiops_env)
...
if [ "$1" = "--env" ]; then
    echo KHIOPS_PATH "$KHIOPS_PATH"
    echo KHIOPS_COCLUSTERING_PATH "$KHIOPS_COCLUSTERING_PATH"
    echo KHIOPS_MPI_COMMAND "$KHIOPS_MPI_COMMAND"
    echo KHIOPS_JAVA_PATH "$KHIOPS_JAVA_PATH"
    echo KHIOPS_CLASSPATH "$KHIOPS_CLASSPATH"
    echo KHIOPS_LAST_RUN_DIR "$KHIOPS_LAST_RUN_DIR"
    echo KHIOPS_PROC_NUMBER "$KHIOPS_PROC_NUMBER"
    echo KHIOPS_TMP_DIR "$KHIOPS_TMP_DIR"
    echo KHIOPS_MEMORY_LIMIT "$KHIOPS_MEMORY_LIMIT"
    echo KHIOPS_API_MODE "$KHIOPS_API_MODE"
    echo KHIOPS_RAW_GUI "$KHIOPS_RAW_GUI"
    echo KHIOPS_DRIVERS_PATH "$KHIOPS_DRIVERS_PATH"
    echo KHIOPS_JAVA_ERROR "$KHIOPS_JAVA_ERROR"
    echo KHIOPS_MPI_ERROR "$KHIOPS_MPI_ERROR"
    echo KHIOPS_MPI_VERBOSE "$KHIOPS_MPI_VERBOSE"
    
	# Additional variables for MPI
	echo OMPI_MCA_btl_vader_single_copy_mechanism none
fi

```

- on Rocky 9 (Docker):
```bash
$ cat $(which khiops_env)
...
if [ "$1" = "--env" ]; then
    echo KHIOPS_PATH "$KHIOPS_PATH"
    echo KHIOPS_COCLUSTERING_PATH "$KHIOPS_COCLUSTERING_PATH"
    echo KHIOPS_MPI_COMMAND "$KHIOPS_MPI_COMMAND"
    echo KHIOPS_JAVA_PATH "$KHIOPS_JAVA_PATH"
    echo KHIOPS_CLASSPATH "$KHIOPS_CLASSPATH"
    echo KHIOPS_LAST_RUN_DIR "$KHIOPS_LAST_RUN_DIR"
    echo KHIOPS_PROC_NUMBER "$KHIOPS_PROC_NUMBER"
    echo KHIOPS_TMP_DIR "$KHIOPS_TMP_DIR"
    echo KHIOPS_MEMORY_LIMIT "$KHIOPS_MEMORY_LIMIT"
    echo KHIOPS_API_MODE "$KHIOPS_API_MODE"
    echo KHIOPS_RAW_GUI "$KHIOPS_RAW_GUI"
    echo KHIOPS_DRIVERS_PATH "$KHIOPS_DRIVERS_PATH"
    echo KHIOPS_JAVA_ERROR "$KHIOPS_JAVA_ERROR"
    echo KHIOPS_MPI_ERROR "$KHIOPS_MPI_ERROR"
    echo KHIOPS_MPI_VERBOSE "$KHIOPS_MPI_VERBOSE"
    
	# Additional variables for MPI
	echo OMPI_MCA_btl_vader_single_copy_mechanism none
	echo PSM3_DEVICES self
fi
```

Reply:
the problem it that the indentation in khiops_env is done with 4 spaces. So I changed all `\t` chars in install.cmake to `   ` and the is no more problem

Reply:
Ok.